### PR TITLE
Wk 163 roles

### PR DESCRIPTION
**Pull request by aditudorache**
This PR adds basic role functionality and navigation.

### CODE REVIEW

File: src/views/manage/ManageLocationPage.jsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import BlueLinkButton from 'shared/buttons/BlueLinkButton';
+import { appRoutes } from 'constants.js';
+import Footer from '../../components/footer/Footer';
+import ManageLocationPageStyle, {
+  ContentStyle,
+} from './ManageLocationPageStyle';
```
Comment (jasperswart) [inline src/views/manage/ManageLocationPage.jsx]: waarom de styles in aparte file?

Comment (aditudorache) [reply src/views/manage/ManageLocationPage.jsx]: for readability

File: src/components/map/hooks/useMap.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { useRef, useEffect } from 'react';
+import L from 'leaflet';
```
Comment (jasperswart) [inline src/components/map/hooks/useMap.js]: Leaflet does not seem to be used

File: src/views/manage/MangeLocationPage.test.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```
Comment (jasperswart) [inline src/views/manage/MangeLocationPage.test.js]: typo filename: MangeLocationPage.test.js

File: src/components/map/hooks/useMap.js (hunk 2 above)
Comment (aditudorache) [reply src/components/map/hooks/useMap.js]: is used on line 22: ` L.tileLayer`

File: src/views/manage/MangeLocationPage.test.js (hunk 3 above)
Comment (aditudorache) [reply src/views/manage/MangeLocationPage.test.js]: thanks!

**Pull request merged**